src/components/site-runners.tsx

```diff
@@ -47,28 +53,33 @@ export function RunnerProvider({
   )
 }
 
+/**
+ * Handles the list of sites, and functions to add and remove them
+ */
 export function useSiteRunners(): {
   sites: Map<string, GatsbySite>
-  addSite: (path: string) => GatsbySite | undefined
+  addSite: (siteInfo: ISiteInfo) => GatsbySite | undefined
 } {
   const { sites, addSite: addSiteToContext } = useContext(RunnerContext)
 
   const addSite = useCallback(
-    (root: string): GatsbySite | undefined => {
-      if (sites?.has(root)) {
-        console.log(`got one`)
-        return sites.get(root)
+    (siteInfo: ISiteInfo): GatsbySite | undefined => {
+      if (sites?.has(siteInfo.path)) {
+        console.log(`Already got one`)
```

Comment:
Is this preceded by another log? It's a bit non-descriptive (albeit arguably less non-descriptive after this change) on its own :)

Reply:
These are just for me right now – they're not yet exposed to the user